modules/odo-new-commands-30.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/developer_cli_odo/understanding-odo.adoc
+
+:_content-type: CONCEPT
+[id="odo-new-commands-30_{context}"]
+= New commands in odo 3.0
+
+`odo` 3.0 includes new commands to improve the developer experience, providing the same functionality as before, but in a faster and more efficient manner.
+
+== odo init
+
+The `odo init` command automatically analyzes your directory and, based on the project type, suggests an appropriate Devfile to use. It also provides a better interactive experience for configuring your Devfile according to your project requirement. With this feature, you can set the environment variables exported to your application and the ports exposed by it on the cluster.
+
+== odo dev
+
+The `odo dev` command allows you to code, build, run, debug and test your applications on the cluster in a continuous workflow. It combines the `odo push` and `odo watch` commands from `odo` version 2 into a single command.
+
+Continuous workflow:: `odo dev` is a non-terminating command that runs until you enter `Ctrl+C`. It continuously watches the directory for any changes (including the changes occurring in the Devfile) and automatically synchronizes them with the application running on the cluster.
+
+Manual trigger to apply local changes on the cluster:: By default, `odo dev` automatically detects any changes in the directory, and synchronizes with the application running on the cluster. However, it is possible to trigger manually by running the command with the `--no-watch` flag and pressing `p` to trigger the synchronization.
+
+Improved watch system:: The `odo dev` command continuously monitors the resources created to run the application on the cluster and ensures that the application always stays running.
+
+Resource cleanup:: Before exiting, the `odo dev` command cleans up any resources that it created for running the application on the cluster.
+
+Port forwarding to access the application:: The `odo dev` command uses port-forwarding for application access, making it much faster to access the application.
+
+== odo add binding
+
+The `odo add binding` command allows the application to connect to an Operator-backed service and replaces the `odo link` command.
+
+Interactive mode:: The `odo add binding` command improves the developer experience by providing an interactive mode that lists bindable service in all the namespaces, and providing a way to personalize related configuration.
```

Comment:
s/service/services
s/ and providing a way to personalize/and the mechanism to personalize - ? Trying to avoid the repetition of - 'providing'